webapp/shop/advantage/views.py

```diff
@@ -72,6 +72,13 @@ def pro_page_view(advantage_mapper, **kwargs):
 def advantage_view(advantage_mapper, is_in_maintenance, **kwargs):
     is_technical = False
     try:
+        channel_account = advantage_mapper.get_purchase_account(
+            "canonical-pro-channel"
+        )
+        if channel_account:
+            return flask.render_template(
+                "account/forbidden.html", reason="channel_account"
```

Comment:
I thought displaying a forbidden page with a link to the distributor page was better than redirecting to the distributor page immediately but please let me know if the other way is better!

Reply:
I think this is a good approach.